tools/scrapd-merger.py

```diff
@@ -205,6 +212,7 @@ def test_merge_00(self):
         "Case": "17-0770668",
         "Date": "March 18, 2017",
         "Fatal crashes this year": "13",
+        "ID": "17-0770668-1",
         "Link": "http://austintexas.gov/news/traffic-fatality-13-2",
         "Location": "4200 block of Southwest Parkway",
         "Time": "8:07 a.m."
```

Comment:
At least one test covering a case with multiple fatalities should be added.

Reply:
I added _test_update_two_entries_from_same_case_with_ID_field_.